cmd/builder/fake_command.go

```diff
@@ -43,3 +49,19 @@ func (f *FakeCommand) Exec(args ...string) (outStr string, err error) {
 	f.ArgsExec = args
 	return
 }
+
+// Interactive will send the command to an interactive execution.
+func (f *FakeFailedCommand) Interactive(args ...string) (err error) {
+	f.CalledInteractive = true
```

Comment:
here we can reuse the logic already implemented: `f.FakeCommand.Interactive(args...)` so we don't repeat it!